x-pack/plugin/core/src/main/java/org/elasticsearch/xpack/core/security/authz/privilege/SystemPrivilege.java

```diff
@@ -40,6 +40,7 @@ public final class SystemPrivilege extends Privilege {
         RetentionLeaseActions.RENEW.name() + "*", // needed for CCR to renew retention leases
         "indices:admin/settings/update", // needed for DiskThresholdMonitor.markIndicesReadOnly
         CompletionPersistentTaskAction.NAME, // needed for ShardFollowTaskCleaner
+        "cluster:admin/migration/update_version", // needed for UpdateIndexMigrationVersionAction
```

Comment:
Do we really need this here (we might I'm really not sure, just asking)?

Reply:
The call to trigger the `UpdateIndexMigrationVersionAction` is done as the `_system` user, with the `_system` role that has the `SystemPrivilege`. The privilege allowed actions are defined here. I ran into issues (`_system` is not allowed to...). I'll investigate if it fits better somewhere else. Thanks for raising this!